lang/en/validation.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'accepted'             => 'The :attribute must be accepted.',
```

Comment:
```suggestion
    'accepted' => 'The :attribute must be accepted.',
```
fix tương tự các line khác cho m nha

Reply:
Em đã sửa ạ